doc/modules/permutation_importance.rst

```diff
@@ -64,15 +74,44 @@ highlight which features contribute the most to the generalization power of the
 inspected model. Features that are important on the training set but not on the
 held-out set might cause the model to overfit.
 
-.. warning::
-
-  Features that are deemed of **low importance for a bad model** (low
-  cross-validation score) could be **very important for a good model**.
-  Therefore it is always important to evaluate the predictive power of a model
-  using a held-out set (or better with cross-validation) prior to computing
-  importances. Permutation importance does not reflect to the intrinsic
-  predictive value of a feature by itself but **how important this feature is
-  for a particular model**.
+The permutation feature importance is the decrease in a model score when a single
+feature value is randomly shuffled. The score function to be used for the
+computation of importances can be specified with the `scoring` argument,
+which also accepts multiple scorers. Using multiple scorers is more computationally
+efficient than sequentially calling :func:`permutation_importance` several times
+with a different scorer, as it reuses model predictions.
+Different metrics might lead to significantly different feature importances.
+An example of using multiple scorers is shown below, employing a list of metrics,
+but more input formats are possible, as documented in
+`multimetric-scoring <https://scikit-learn.org/stable/modules/model_evaluation.html#multimetric-scoring>`_.
```

Comment:
Internal links should use references to avoid hardcoding the base URL:

```
but more input formats are possible, as documented in :ref:`multimetric_scoring`.
```

 should work in this case I believe.